translations/ja.po

```diff
@@ -801,75 +801,75 @@ msgstr ""
 #: src\commands\yugipedia.ts:38
 msgctxt "command-option"
 msgid "page"
-msgstr ""
+msgstr "ページ"
 
 #: src\commands\deck.ts:122
 msgctxt "command-option"
 msgid "url"
-msgstr ""
+msgstr "URL"
```

Comment:
Discord rejects uppercase characters for this.

Reply:
Make sure your result makes sense. This is for the _subcommand name_. In English, it was `/deck url`, so now in Japanese, it will be `/[localization for "deck"] [localization for "url"]`.

Reply:
maybe I'll revert it to `url` XD